src/Compilers/CSharp/Portable/BoundTree/Formatting.cs

```diff
@@ -165,4 +165,9 @@ internal partial class BoundPassByCopy
     {
         public override object Display => Expression.Display;
     }
+
+    internal partial class BoundUnconvertedAddressOfOperator
+    {
+        public override object Display => $"&{Operand.Display}";
```

Comment:
>$"&{Operand.Display}" [](start = 42, length = 21)

I think we don't want to build the string right on the spot here, see examples of usage of FormattableStringFactory above.

Reply:
Especially that we are localizing display for BoundMethodGroup

---
In reply to: [394001595](https://github.com/dotnet/roslyn/pull/42488#discussion_r394001595) [](ancestors = 394001595)